libs/ui-watt/src/lib/components/input/date-range-input/watt-date-range-input.component.ts

```diff
@@ -32,13 +32,18 @@ import {
 import { BooleanInput, coerceBooleanProperty } from '@angular/cdk/coercion';
 import { ControlValueAccessor, NgControl } from '@angular/forms';
 import { FormatWidth, getLocaleDateFormat } from '@angular/common';
+import { MatEndDate, MatStartDate } from '@angular/material/datepicker';
 import { MatFormFieldControl } from '@angular/material/form-field';
-import { Subject, takeUntil } from 'rxjs';
+import { combineLatest, map, Subject, takeUntil, tap } from 'rxjs';
+import { parse } from 'date-fns';
+import { formatInTimeZone } from 'date-fns-tz';
 
 import { WattInputMaskService } from '../shared/watt-input-mask.service';
 import { WattRangeInputService } from '../shared/watt-range-input.service';
 import { WattRange } from '../shared/watt-range';
 
+const dateTimeFormat = 'dd-MM-yyyy';
```

Comment:
Suggestion: This should properly be consolidated with `getInputFormat` to support localization, but we can do this in #420.

Reply:
Agree. Let's look at it when working on that issue.